components/dashboard/src/settings/Teams.tsx

```diff
@@ -527,6 +528,16 @@ function AllTeams() {
                 <div className="flex-grow ">
                     <h3 className="self-center">All Teams</h3>
                     <h2>Manage teams and team members.</h2>
+                    <ItemsList className="pt-2">
```

Comment:
**question**: Not sure if this needs to be added also below [`L431`](https://github.com/gitpod-io/gitpod/blob/0e9697138896d4451e48f4fd2bb91eb37282cdbf/components/dashboard/src/settings/Teams.tsx#L431) to make it visible at all times. Cc @AlexTugarev